index.js

```diff
@@ -159,59 +170,37 @@ function printResultsForCommand(subjectName, subjectResults, command) {
 // Keep track of elapsed time
 const processStartTime = performance.now();
 
-// Initialize the Cypress runner configuration
-let cypressConfig = {};
-
-try {
-  // Passing `--configFile false` will explicitly _not_ attempt to load a configuration file
-  if (argv.configFile !== 'false') {
-    cypressConfig = JSON.parse(fs.readFileSync(path.resolve(process.cwd(), argv.configFile), 'utf8')) || {};
-  }
-} catch (err) {
-  if (err.code === 'ENOENT') {
-    console.warn(`
-      Could not load a Cypress configuration at path: \`${argv.configFile}\`
-
-      Specify the path to your configuration with the \`--configFile\` command-line option,
-      or run this command from the appropriate working directory.`.replace(/  +/g, '')
-    );
-  } else {
-    console.error(err)
-  }
-}
-
-// Fall-back to Cypress configuration option if CLI option is not specified, otherwise use the default value
-argv.testFiles = argv.testFiles ?? cypressConfig.testFiles ?? '**/*.*';
-
-// Fall-back to `integrationFolder` option if it doesn't exist in the config
-if (cypressConfig.integrationFolder === undefined) {
-  cypressConfig.integrationFolder = argv.integrationFolder;
-}
+// Verify if the testFiles argv exists, otherwise use the default value
+argv.testFiles = argv.testFiles || '**/*.*';
 
 // Read user-specified spec files from filesystem
 let specFiles;
 
 argv.fileIdentifiers = castArray(argv.fileIdentifiers);
-
 try {
-  specFiles = glob.sync(cypressConfig.integrationFolder + '/' + argv.testFiles)
+  specFiles = glob.sync(argv.specPattern + '/' + argv.testFiles)
     .filter(fileName => {
       return (
         argv.fileIdentifiers.some((fileIdentifier) => fileName.toLowerCase().includes(fileIdentifier))
       );
-    })
+    });
+  if (argv.excludeSpecPattern.length > 0) {
```

Comment:
this could fail if `argv.excludeSpecPattern` is undefined

Reply:
I'm not sure because we're including a default value `[]` in the excludeSpecPattern. May I include the validation anyway?

Reply:
You're right, the code change in [58d39db](https://github.com/trentrand/cypress-utils/pull/25/commits/58d39db353ae1decd42a55df87927648b18d2b67) fixes this